backend/scripts/docker/postgres/postgres_server.sh

```
@@ -0,0 +1,92 @@
+k#!/bin/bash
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)